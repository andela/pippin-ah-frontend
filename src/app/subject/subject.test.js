import React from 'react';
import { shallow } from 'enzyme';
import { Subject } from './index';

describe(' Component', () => {
  it('should render the Subject component', () => {
    const component = shallow(<Subject />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
