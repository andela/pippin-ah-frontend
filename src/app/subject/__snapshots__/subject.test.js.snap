// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Component should render the Subject component 1`] = `
<Fragment>
  <div
    className="container"
  >
    <div
      className="col s12 center-align"
    >
      <h3>
        Arts
      </h3>
    </div>
    <div
      className="row"
    >
      <div
        className="col s12 m6 l4"
      >
        <div
          className="card"
        >
          <div
            className="card-image"
          >
            <img
              alt="Cover"
              className="activator"
              src="https://i.pinimg.com/564x/b2/69/9d/b2699de28deffa0883978ffecfd977a1.jpg?b=t"
            />
          </div>
          <div
            className="card-content right-align"
          >
            <span
              className="card-title activator flow-text truncate center-align"
            >
              How to Develop Your Ideas
            </span>
            <strong>
              by: Bugs Burney
            </strong>
            <div>
              <a
                href="/#"
              >
                Read more
              </a>
            </div>
          </div>
          <div
            className="card-reveal"
          >
            <span
              className="card-title grey-text text-darken-4"
            >
              How to Develop Your Ideas
              <i
                className="material-icons right"
              >
                close
              </i>
            </span>
            <p>
              This article explains how I learnt to share my ideas through art.
            </p>
            <div>
              <a
                href="/#"
              >
                Click here to read
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s12 m6 l4"
      >
        <div
          className="card"
        >
          <div
            className="card-image"
          >
            <img
              alt="Cover"
              className="activator"
              src="https://i.pinimg.com/564x/b2/69/9d/b2699de28deffa0883978ffecfd977a1.jpg?b=t"
            />
          </div>
          <div
            className="card-content right-align"
          >
            <span
              className="card-title activator flow-text truncate center-align"
            >
              How to Develop Your Ideas
            </span>
            <strong>
              by: Bugs Burney
            </strong>
            <div>
              <a
                href="/#"
              >
                Read more
              </a>
            </div>
          </div>
          <div
            className="card-reveal"
          >
            <span
              className="card-title grey-text text-darken-4"
            >
              How to Develop Your Ideas
              <i
                className="material-icons right"
              >
                close
              </i>
            </span>
            <p>
              This article explains how I learnt to share my ideas through art.
            </p>
            <div>
              <a
                href="/#"
              >
                Click here to read
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s12 m6 l4"
      >
        <div
          className="card"
        >
          <div
            className="card-image"
          >
            <img
              alt="Cover"
              className="activator"
              src="https://i.pinimg.com/564x/b2/69/9d/b2699de28deffa0883978ffecfd977a1.jpg?b=t"
            />
          </div>
          <div
            className="card-content right-align"
          >
            <span
              className="card-title activator flow-text truncate center-align"
            >
              How to Develop Your Ideas
            </span>
            <strong>
              by: Bugs Burney
            </strong>
            <div>
              <a
                href="/#"
              >
                Read more
              </a>
            </div>
          </div>
          <div
            className="card-reveal"
          >
            <span
              className="card-title grey-text text-darken-4"
            >
              How to Develop Your Ideas
              <i
                className="material-icons right"
              >
                close
              </i>
            </span>
            <p>
              This article explains how I learnt to share my ideas through art.
            </p>
            <div>
              <a
                href="/#"
              >
                Click here to read
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s12 m6 l4"
      >
        <div
          className="card"
        >
          <div
            className="card-image"
          >
            <img
              alt="Cover"
              className="activator"
              src="https://i.pinimg.com/564x/b2/69/9d/b2699de28deffa0883978ffecfd977a1.jpg?b=t"
            />
          </div>
          <div
            className="card-content right-align"
          >
            <span
              className="card-title activator flow-text truncate center-align"
            >
              How to Develop Your Ideas
            </span>
            <strong>
              by: Bugs Burney
            </strong>
            <div>
              <a
                href="/#"
              >
                Read more
              </a>
            </div>
          </div>
          <div
            className="card-reveal"
          >
            <span
              className="card-title grey-text text-darken-4"
            >
              How to Develop Your Ideas
              <i
                className="material-icons right"
              >
                close
              </i>
            </span>
            <p>
              This article explains how I learnt to share my ideas through art.
            </p>
            <div>
              <a
                href="/#"
              >
                Click here to read
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s12 m6 l4"
      >
        <div
          className="card"
        >
          <div
            className="card-image"
          >
            <img
              alt="Cover"
              className="activator"
              src="https://i.pinimg.com/564x/b2/69/9d/b2699de28deffa0883978ffecfd977a1.jpg?b=t"
            />
          </div>
          <div
            className="card-content right-align"
          >
            <span
              className="card-title activator flow-text truncate center-align"
            >
              How to Develop Your Ideas
            </span>
            <strong>
              by: Bugs Burney
            </strong>
            <div>
              <a
                href="/#"
              >
                Read more
              </a>
            </div>
          </div>
          <div
            className="card-reveal"
          >
            <span
              className="card-title grey-text text-darken-4"
            >
              How to Develop Your Ideas
              <i
                className="material-icons right"
              >
                close
              </i>
            </span>
            <p>
              This article explains how I learnt to share my ideas through art.
            </p>
            <div>
              <a
                href="/#"
              >
                Click here to read
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s12 m6 l4"
      >
        <div
          className="card"
        >
          <div
            className="card-image"
          >
            <img
              alt="Cover"
              className="activator"
              src="https://i.pinimg.com/564x/b2/69/9d/b2699de28deffa0883978ffecfd977a1.jpg?b=t"
            />
          </div>
          <div
            className="card-content right-align"
          >
            <span
              className="card-title activator flow-text truncate center-align"
            >
              How to Develop Your Ideas
            </span>
            <strong>
              by: Bugs Burney
            </strong>
            <div>
              <a
                href="/#"
              >
                Read more
              </a>
            </div>
          </div>
          <div
            className="card-reveal"
          >
            <span
              className="card-title grey-text text-darken-4"
            >
              How to Develop Your Ideas
              <i
                className="material-icons right"
              >
                close
              </i>
            </span>
            <p>
              This article explains how I learnt to share my ideas through art.
            </p>
            <div>
              <a
                href="/#"
              >
                Click here to read
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</Fragment>
`;
